test/test.py

```diff
@@ -4,32 +4,62 @@
 import subprocess
 import sys
 
+if "CRUNCH_BUILD_DIR" in os.environ.keys():
+    build_dir = os.environ["CRUNCH_BUILD_DIR"]
+else:
```

Comment:
These 4 lines can be written as `build_dir = os.environ.get("CRUNCH_BUILD_DIR")`

Reply:
I used `os.getenv("CRUNCH_BUILD_DIR", "build")` instead.